emqx/manifest.json

```
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fa40ec7e-e8f6-4c4b-a675-31716b23a9df",
+  "app_id": "emqx",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect performance, health data, message throughput and essage latency on MQTT brokers, and more.",
```

```suggestion
    "description": "Collect data on the performance, health, message throughput, and message latency of MQTT brokers",
```
Feel free to revise if this is not accurate